backend/mora/service/addresses/dar.py

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright (c) 2017-2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import requests
+import flask
+from . import base
```

Comment:
Insert a newline between the 3rd party modules and our own. And perhaps sort them alphabetically?
```suggestion

from . import base
```